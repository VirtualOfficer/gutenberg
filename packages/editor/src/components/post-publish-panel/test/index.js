/**
 * External dependencies
 */
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import { PostPublishPanel } from '../index';

describe( 'PostPublishPanel', () => {
	it( 'should render the pre-publish panel if the post is not saving, published or scheduled', () => {
		const wrapper = shallow(
			<PostPublishPanel
				isPublished={ false }
				isScheduled={ false }
				isSaving={ false }
			/>
		);
		expect( wrapper ).toMatchSnapshot();
	} );
} );
