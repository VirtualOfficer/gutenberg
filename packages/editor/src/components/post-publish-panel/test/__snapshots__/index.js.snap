// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostPublishPanel should render the pre-publish panel if the post is not saving, published or scheduled 1`] = `
<div
  className="editor-post-publish-panel"
  isSaving={false}
>
  <div
    className="editor-post-publish-panel__header"
  >
    <div
      className="editor-post-publish-panel__header-publish-button"
    >
      <WithSelect(WithDispatch(PostPublishButton))
        focusOnMount={true}
        onSubmit={[Function]}
      />
      <span
        className="editor-post-publish-panel__spacer"
      />
    </div>
    <IconButton
      aria-expanded={true}
      icon="no-alt"
      label="Close panel"
    />
  </div>
  <div
    className="editor-post-publish-panel__content"
  >
    <WithSelect(PostPublishPanelPrepublish) />
  </div>
  <div
    className="editor-post-publish-panel__footer"
  >
    <WithInstanceId(CheckboxControl)
      label="Always show pre-publish checks."
    />
  </div>
</div>
`;
